rye/src/cli/publish.rs

```diff
@@ -21,10 +21,10 @@ use crate::utils::{get_venv_python_bin, CommandOutput};
 pub struct Args {
     /// The distribution files to upload to the repository (defaults to <workspace-root>/dist/*).
```

Comment:
And optionally we could even do
```suggestion
    /// The distribution files to upload to the repository [default: <workspace-root>/dist/*].
```
Though this is untested.